vector/v.voronoi/skeleton.c

```diff
@@ -525,7 +525,7 @@ int tie_up(void)
 	                         IPoints[i]);
 	}
 
-	distmin = 1. / 0.; /* +inf */
+	distmin = INFINITY; /* +inf */
```

Comment:
This change is not Cmake related and should be separated out

(into a new INFINITY related PR)